opm/parser/eclipse/EclipseState/Schedule/Tuning.hpp

```diff
@@ -22,233 +22,92 @@
-        const DynamicState<double>& getXXXTTE() const;
-        const DynamicState<double>& getXXXCNV() const;
-        const DynamicState<double>& getXXXMBE() const;
-        const DynamicState<double>& getXXXLCV() const;
-        const DynamicState<double>& getXXXWFL() const;
-        const DynamicState<double>& getTRGFIP() const;
-        const DynamicState<double>& getTRGSFT() const;
-        const DynamicState<int>& getTRGSFT_has_value() const;
-        const DynamicState<double>& getTHIONX() const;
-        const DynamicState<int>& getTRWGHT() const;
-
-        const DynamicState<int>& getNEWTMX() const;
-        const DynamicState<int>& getNEWTMN() const;
-        const DynamicState<int>& getLITMAX() const;
-        const DynamicState<int>& getLITMIN() const;
-        const DynamicState<int>& getMXWSIT() const;
-        const DynamicState<int>& getMXWPIT() const;
-        const DynamicState<double>& getDDPLIM() const;
-        const DynamicState<double>& getDDSLIM() const;
-        const DynamicState<double>& getTRGDPR() const;
-        const DynamicState<double>& getXXXDPR() const;
-        const DynamicState<int>& getXXXDPR_has_value() const;
-
-        const std::map<std::string,bool>& getResetValues() const;
-
-        bool operator==(const Tuning& data) const;
-
-    private:
-        /* Record1 */
-        DynamicState<double> m_TSINIT;
-        DynamicState<double> m_TSMAXZ;
-        DynamicState<double> m_TSMINZ;
-        DynamicState<double> m_TSMCHP;
-        DynamicState<double> m_TSFMAX;
-        DynamicState<double> m_TSFMIN;
-        DynamicState<double> m_TSFCNV;
-        DynamicState<double> m_TFDIFF;
-        DynamicState<double> m_THRUPT;
-        DynamicState<double> m_TMAXWC;
-        DynamicState<int>    m_TMAXWC_has_value;
-        /* Record 2 */
-        DynamicState<double> m_TRGTTE;
-        DynamicState<double> m_TRGCNV;
-        DynamicState<double> m_TRGMBE;
-        DynamicState<double> m_TRGLCV;
-        DynamicState<double> m_XXXTTE;
-        DynamicState<double> m_XXXCNV;
-        DynamicState<double> m_XXXMBE;
-        DynamicState<double> m_XXXLCV;
-        DynamicState<double> m_XXXWFL;
-        DynamicState<double> m_TRGFIP;
-        DynamicState<double> m_TRGSFT;
-        DynamicState<int>    m_TRGSFT_has_value;
-        DynamicState<double> m_THIONX;
-        DynamicState<int>    m_TRWGHT;
-        /* Record 3 */
-        DynamicState<int>    m_NEWTMX;
-        DynamicState<int>    m_NEWTMN;
-        DynamicState<int>    m_LITMAX;
-        DynamicState<int>    m_LITMIN;
-        DynamicState<int>    m_MXWSIT;
-        DynamicState<int>    m_MXWPIT;
-        DynamicState<double> m_DDPLIM;
-        DynamicState<double> m_DDSLIM;
-        DynamicState<double> m_TRGDPR;
-        DynamicState<double> m_XXXDPR;
-        DynamicState<int>    m_XXXDPR_has_value;
-        std::map<std::string, bool> m_ResetValue;
+    struct Tuning {
+        // Record1
+        double TSINIT = 1.0 * Metric::Time;
```

Comment:
My preferred way to do this is via the default value embedded in the generated keywords:
```
using TuningKW = ParserKeywords::TUNING;
double TSINIT = TuningKW::TSINIT::defaultValue * Metric::Time;
...
```
but I realize it is quite verbose and not as much used. Will not require anything.

Reply:
done.